osu.Game/Overlays/MusicController.cs

```diff
@@ -40,6 +40,11 @@ public partial class MusicController : CompositeDrawable
         /// </summary>
         public bool UserPauseRequested { get; private set; }
 
+        /// <summary>
+        /// Whether control of the global track should be allowed.
+        /// </summary>
+        public readonly BindableBool AllowTrackControl = new BindableBool(true);
```

Comment:
Also this is really about _user_ control of the global track. As the implementation shows below, screens are still able to manipulate the global track if they need, it's just not via user request. Dunno, probably a matter of xmldoc and/or rename at most again.

Reply:
Adjusted xmldoc.